### ISSUE TITLE
Mark UWP6.0 packages as stable

### ISSUE THREAD

**Issue by joshfree** [dotnet/corefx#23355]
Tracking issue for marking UWP6.0 as stable

**Comment by joperezr** [comment]
FYI: @weshaggard @ericstj 

I'll take care of this once we are done with the brokered files DCR and we are ready to call a build the RTM one.

**Comment by weshaggard** [comment]
FYI: We haven't been marking the transport/private packages as stable as we don't push them to nuget.org. So for this release I expect only the Microsoft.NETCore.UniversalWindowsPlatform package to be marked stable in core-setup. We might need to stablize Microsoft.NETCore.Platforms/Targets as well but hopefully not as I don't think there are any UWP specific changes in them.

**Comment by JimSEOW** [comment]
If there a way to have access to the a pre-release version of UWP 6.0? The library I am working works WELL with .NET standard 1.6 [_which is not compatible with UWP_] and I would like to bring that to **UWP IoT Core Creative Update.**

**Comment by joshfree** [comment]
@JimSEOW I encourage you to join the Preview program: [Early Access to Visual Studio Preview](https://www.visualstudio.com/vs/preview/).

**Comment by JimSEOW** [comment]
@joshfree Just installed15.3.1. Is there a Nuget Source where I could have access to the preview version of UWP6.0? Hope UWP6.0 will be released on time end of this month.

**Comment by joshfree** [comment]
@JimSEOW I'll update you on this issue once it's available through the Visual Studio Preview channel.

**Comment by JimSEOW** [comment]
@joshfree Thank you, The combination of Xamarin Forms 3.0  (embedding) using .NET standard 2.0 (with hopefully soon XAML standard) and UWP6.0 is critical for our corporate customers. Great jobs to the team advancing UWP6.0 and beyond.

**Comment by JimSEOW** [comment]
@joshfree  Where can I find the diff between .NET standard 2.X and 2.0? I would like to vote for more System.Drawing.Drawing and System.Drawing.Drawing2D to go into .NET standard 2.X.

**Comment by joperezr** [comment]
we haven't finilized defining anything above 2.0 yet that I'm aware, but the right place to make suggestion like that would be on https://github.com/dotnet/standard. cc: @mellinoe who owns System.Drawing.

**Comment by JimSEOW** [comment]
@joshfree  Who and where could I feedback on **Cross function unmet needs** involving .NET Standard 2.0, UWP6.0 and beyond and XAML standard?

We need to map out the future path of UWP XAML interop for 3D - which is critical for **democratizing** _Microsoft Mixed Reality and Remix 3D_ to C# community.

I have checked with @mellinoe, he is making good progress towards cross platform OpenGL for .NETStandard 2.0 (Amazing).  He needs someone from XAML standard AND >= UWP6.1 to map out the future path to address the need for **UWP XAML Interop for 3D for not only CLR/C++ BUT c# communities. 

FYI: Is there someone from Microsoft who has interest of serving SharpDX community for not only the CLR/C++ BUT c# communities ON XAML Interop?

**Comment by JimSEOW** [comment]
Waiting for UWP6.0 to test [`Xamarin Forms and EF Core 2.0`](https://github.com/adamped/EFXamarin/issues/1#issue-252243059)

**Comment by migueldeicaza** [comment]
@JimSEOW While it does not currently use XAML, or is slated for some of standardized effort, we are distributing a a 3D graphics library for retained graphics, that is similar in spirit to what you want.

The library is [UrhoSharp](https://developer.xamarin.com/guides/cross-platform/urho/introduction/)  a cross-platform .NET binding that supports every platform you might want to target and in particular works great with HoloLens and other MixedReality headsets and *also* works with both iOS's new Augmented Reality APIs (ARKit) [sample](https://developer.xamarin.com/guides/ios/platform_features/introduction-to-ios11/arkit/urhosharp/).

What is nice about UrhoSharp is that you can reuse the same code across UWP, Android, iOS, Mac and Windows for 3D graphics and supports both HoloLens and ARKit for creating AR and VR apps.

Here is a demo showing some animated 3D characters rendered with iOS ARKit responding to user input and navigating a real-life terrain (Egor's apartment):

https://www.youtube.com/watch?v=wNrLx5A0vkc

Source code:

https://github.com/EgorBo/ARKitXamarinDemo

**Comment by JimSEOW** [comment]
@migueldeicaza I am an avid follower AND believer of what is happening to UrhoSharp in HoloLens and ARKit. I have been lobbying for the Urho3D c++ community to support VisualStudio ARM C++ build since [2016 ](https://discourse.urho3d.io/t/windows-10-uwp-support/1659)[ => 2017](https://discourse.urho3d.io/t/will-urho3d-run-native-with-windows-10-on-arm-q42017/3139) so we could **COMPLETE the Full Cross Platform destiny of UrhoSharp** to cover _IoT Core_ and _IoT Core Pro_ (64bit), W10M and more importantly, _W10A_, the Win10 On ARM (ARM Native and not under emulation).

This discussion is motivated by

-  the emerging opportunity triggered by [waves of opportunities and optimisms ](https://github.com/dotnet/designs/issues/12)made possible by **Everything .NET Standard** ( next generation Cross platform PCL[_.NETStandard 2.0_], OneGUI4ALL (inspired by Xamarin Forms) [_XAML Standard_]).

- the **Everything 3D vision/standard** - makes "live and breathe 3D" increasingly part of the Windows UWP ecosystem experience: targeting both corporate and consumer market segments through e.g. Mixed Reality, Remix 3D, View3D, Photos and more!. If we do right, UWP is in the leading position to make 3D integration so pervasive that thinking 3D/4D is the foundation of future education to [cultivate creativity at early age](https://github.com/Microsoft/WindowsTemplateStudio/issues/561).

> [Finding the Primer](https://www.youtube.com/watch?v=HgPSUer1ujM
) [Contact (movie) Jodie Foster, John Hurt]

- the tools to **democratize 3D** and empower more 3D developers to bring more 3D apps to Windows Store. The majority of important REAL WORLD 3D visualization use cases have been ported to WebGL-based frameworks. We need different tools to empower developers to bring these Real World [mostly OpenGL OpenGL ES-based] use cases to **UWP (XAMLStandard)** interOp Apps for the Windows Store. 

_For examples:_
[HoloJSHost ](https://github.com/Microsoft/HoloJS/tree/master/HoloJS/HoloJsHost)of HoloJS makes WebGL to UWP possible. 
We need a version of HoloJSHost that support UWP (XAMLStandard) Interop [we need the [3D version](https://stackoverflow.com/questions/39290861/how-to-add-3d-context-works-in-jsbridge-by-adding-webglrenderingcontext) of UWP (XAML) for 2D using [JsBridge](https://github.com/deltakosh/JsBridge)]
 
In summary:
Much of the Real World 3D use cases are based on OpenGL/OpenGL ES and we need tools to empower developers to port these existing codes to UWP(XAML Standard) [e.g. [adding new pages to WindowsTemplateStudio](https://github.com/Microsoft/WindowsTemplateStudio) ] to make 3D Everywhere throughout the UWP ecosystem, especially to stimulate children at early age to express what they understand the world around them OUTDOOR using e.g. Windows AR with friends.

**Comment by JimSEOW** [comment]
Thank YOU ALL!

Xamarin FORM with EF CORE 2.0 nicely compiled with UWP6.0!

![2017-08-25_12-46-58](https://user-images.githubusercontent.com/8244633/29711106-aa92954a-8993-11e7-890d-321d912526c7.png)

**Comment by JimSEOW** [comment]
Feedback!

[This discussion ](https://github.com/sharpdx/SharpDX/issues/864#issuecomment-324942224)reflects there is need to do more to disseminate the impact of .NETStandard 2.0 and UWP 6.0

![2017-08-25_12-51-25](https://user-images.githubusercontent.com/8244633/29719450-57fc744a-89b6-11e7-9f10-0242e9c51d6c.png)
![2017-08-25_16-45-03](https://user-images.githubusercontent.com/8244633/29719452-5a9d143e-89b6-11e7-969a-32647aa5101d.png)

**Comment by JimSEOW** [comment]
@joshfree    Please confirm:

The latest **UWP6.0preview** which is made available through the insider SDK build **16267** - will only work with

1. W10: fast ring **16275** 
2. W10 IoT Core: fast ring **16267**

NOT 

3. W10M: fast ring **15245**

**Comment by joshfree** [comment]
Closing old issue.  

https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform/6.0.1
